src/cli/vfs-file-stat.ts

```diff
@@ -0,0 +1,22 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@sudhirverma Why does this have Microsoft mentioned in the license header?

Reply:
And do we need a specific file for stats, can't this be directly incorporated in the `virtualfs.ts` file ?

Reply:
So @sudhirverma , we do not use this file anymore?

Reply:
yes, we are not using this file.